armi/reactor/converters/tests/test_uniformMesh.py

```diff
@@ -37,7 +38,10 @@ class TestUniformMeshComponents(unittest.TestCase):
     @classmethod
     def setUpClass(cls):
         random.seed(987324987234)  # so it's always the same
-        cls.o, cls.r = test_reactors.loadTestReactor(TEST_ROOT)
+        cls.o, cls.r = test_reactors.loadTestReactor(
+            TEST_ROOT, customSettings={"xsKernel": "MC2v2"}
```

Comment:
this is alluded to in the commit message. ISOAA has MC2v2 nuclides in it. If you want to do physics you need to specify this setting. Otherwise it crashes. Longterm solution would be to remake ISOAA or just copy some elementals to isotopics in it.

Reply:
Either of those seem reasonable. I think it is fine to ship with artifacts of having run MC2, but not with settings and other things that discuss actually running it. Again, we should be moving closer to begin able to migrate the option itself into the MC2 plugin. Just need to make sure that it's on our path forward